sklearn/datasets/mlcomp.py

```diff
@@ -68,6 +68,11 @@ def load_mlcomp(name_or_id, set_="raw", mlcomp_root=None, **kwargs):
     if not os.path.exists(mlcomp_root):
         raise ValueError("Could not find folder: " + mlcomp_root)
 
+    if name_or_id in ['20news-18828', '20news-19997', '20news-bydate']:
+        raise DeprecationWarning("please consider using "
```

Comment:
Deprecate the whole function with a `@deprecated` decorator. Also mention which version it will be deprecated in (0.19) and which version it will be removed (0.21). Look either at the contributing guidelines or other deprecation messages in the scikit-learn code.